qualification/Jenkinsfile

```
@@ -86,7 +86,7 @@ pipeline {
           * tests and mark the build as unstable.
           */
          sh '''
-           pytest -v -c qualification/pytest-jenkins.ini qualification\
+           spead2_net_raw pytest -v -c qualification/pytest-jenkins.ini qualification\
```

Not really related to your PR, but for my OCD:
```suggestion
           spead2_net_raw pytest -v -c qualification/pytest-jenkins.ini qualification \
```